dartagnan/src/main/java/com/dat3m/dartagnan/Dartagnan.java

```diff
@@ -176,9 +198,8 @@ public static void main(String[] args) throws Exception {
             Configuration solverConfig = Configuration.builder()
                     .setOption(PHANTOM_REFERENCES, valueOf(o.usePhantomReferences()))
                     .build();
-            try (SolverContext ctx = SolverContextFactory.createSolverContext(
+            try (SolverContext ctx = createSolverContext(
                     solverConfig,
-                    BasicLogManager.create(solverConfig),
```

Comment:
Why are you removing this?

TBH, I don't even remember why this is here in the first place.

Reply:
I just moved this line into the function since it depends on the solverConfig anyway. Otherwise, I would need to play around with the type LogManager which is provided by sosy_lab.log and by log4j.